localization/twist_estimator/gyro_odometer/launch/gyro_odometer.launch.xml

```diff
@@ -1,5 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we do this for other packages as well? I haven't seen it yet.

Reply:
I don't think it makes much of a difference, I can remove it if it causes confusion